### PR TITLE
Added home page and dataset

### PR DESCRIPTION
**Pull request by srishti-maurya**
## Added Components

- Card
- Nav 
- MockAPI for mockman

## Added Pages

- Home

## Added dependencies

- react router v6
- react fonts

## UI for reference
UI-min

### CODE REVIEW

File: src/frontend/pages/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+// section collection image
+import bestSellers from "../assets/images/bestsellers.jpeg";
+import newReleases from "../assets/images/new release.jpeg";
+import expertPicks from "../assets/images/expert.jpeg";
+// section genre image
+import contemporaryFiction from "../assets/images/book cluster.jpeg";
+import selfHelp from "../assets/images/self help.jpeg";
+import biographies from "../assets/images/bio.jpeg";
+import spirituality from "../assets/images/spiritual.jpeg";
+import mythology from "../assets/images/history.jpeg";
```
Comment (msdeshmukh009) [inline src/frontend/pages/Home.jsx]: Is there any way to import these images from a single source i.e. [barrel import](https://medium.com/@klauskpm/do-a-barrel-export-aa5b79b76b05)? Cause that will increase the readability of the code.

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,30 @@
+import "./frontend/styles/style.css";
+import MockAPI from "./frontend/components/MockAPI";
+import { useState, useEffect } from "react";
+import { Routes, Route } from "react-router-dom";
+import { Home } from "./frontend/pages/Home";
+export const axios = require("axios").default;
```
Comment (msdeshmukh009) [inline src/App.js]: You can also use  [es6 import/export](https://www.geeksforgeeks.org/difference-between-node-js-require-and-es6-import-and-export/) here. 
`import axios from "axios";`

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good. Nice work!!

**Review by pritam-kr — APPROVED** [review]
All looking good to me 
but You can reduce image size i.e some of the images are 400kb, 500kb. 

- You used React icon which is good.

File: src/frontend/pages/Home.jsx (hunk 1 above)
Comment (srishti-maurya) [reply src/frontend/pages/Home.jsx]: Thank you for your suggestion, I have made the changes.

File: src/App.js (hunk 2 above)
Comment (srishti-maurya) [reply src/App.js]: Thank you for your suggestion, I have made the changes.

**Pull request merged**